docs/rfc/rfc-1.adoc

```diff
@@ -382,14 +382,23 @@ key than the same `signingGroupPubkey` is fraud.
 
 Governable Parameters:
 
+- `redemption_request_timeout`: The amount of time the wallet has to provide
+  redemption proof.
+- `redemption_request_timeout_slashing_amount`: The amount of stake slashed from
+  each member of a wallet for a timed-out redemption request.
+- `redemption_request_timeout_redemeer_bonus_multiplier`: The percentage of the
+  notifier reward from the staking contract the redeemer receives in case of a
+  redemption timeout.
+- `redemption_treasury_fee`: The percentage of redeemed amount put aside as a
+  treasury fee.
 - `wallet_min_btc`: The smallest amount of BTC a wallet can hold before we
   attempt to close the wallet and transfer the funds to a randomly selected
   wallet.
 
 To initiate a redemption, a user with a swept balance > `x` supplies a bitcoin
 address. Then, the system calculates the redemption fee `fee`, and releases an
 amount of bitcoin `y` such that `x = y + fee` to the supplied bitcoin address.
-The remaining `fee` sold by the system to buy back `T` tokens (more about this
+The remaining `fee` is sold by the system to buy back `T` tokens (more about this
```

Comment:
Could we replace `fee` here with `redemption_treasury_fee`?

Reply:
89cfc7d20db222d71cddd56fdc74c2a8f93a964d